app/models/card.rb

```diff
@@ -42,8 +42,9 @@ def next_card
   def assignee_ids=(ids)
     # 1.
     self.touch if self.persisted?
-    # 2.
-    @new_assignees = ids - (assignee_ids + [''])
+    # 2. Convert assignee_ids to string as ids will be an array of
+    # string numbers.
+    @new_assignees = ids - (assignee_ids.map(&:to_s) + [''])
```

Comment:
This change is needed because the ids from the params is an array of strings, i.e. ["", "1", "2", "3"] as it's submitted from the form. I converted the existing assignee ids to strings (which are currently integers) so that the array operation returns the expected results. Previously it was something like ["", "1", "2"] - ([1] + ['']) and is returning ["1", "2"] but not ["2'].